system-test/managed_writer_client_test.ts

```diff
@@ -369,6 +372,68 @@ describe('managedwriter.WriterClient', () => {
     });
   });
 
+  describe('StreamConnection', () => {
+    it('should pass traceId on AppendRequests', async () => {
+      bqWriteClient.initialize();
+      const client = new WriterClient();
+      client.setClient(bqWriteClient);
+
+      // Row 1
+      const row1 = {
+        customer_name: 'Lovelace',
+        row_num: 1,
+      };
+
+      // Row 2
+      const row2 = {
+        customer_name: 'Turing',
+        row_num: 2,
+      };
+
+      try {
+        const connection = await client.createStreamConnection({
+          streamId: managedwriter.DefaultStream,
+          destinationTable: parent,
+        });
+        let writer = new JSONWriter({
+          connection,
+          protoDescriptor,
+        });
+
+        let pw1 = await writer.appendRows([row1, row2]);
+        let pw2 = await writer.appendRows([row1, row2]);
+        await Promise.all([pw1.getResult(), pw2.getResult()]);
+
+        let requests = [pw1.getRequest(), pw2.getRequest()];
+        requests.every(req => {
```

Comment:
Eventually we want only the first request contains the trace id, but I am fine with this as of today.

Reply:
yeah, I'll handle that when I start to work on issue #445, so I only send the full request with schema, traceId and other properties on the first one.